packages/seed-bible/app/sn_components/currentStudyNote.json

```diff
@@ -1,237 +1,110 @@
 [
```

Comment:
@Ahmedki1l can you remove this file so that we're not accidentally including any copyrighted content?